package.json

```diff
@@ -18,57 +18,57 @@
     "build": "electron-webpack && electron-builder"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.8.1",
-    "leaflet": "^1.4.0",
-    "leaflet.offline": "^1.0.4",
+    "@fortawesome/fontawesome-free": "^5.12.1",
+    "leaflet": "^1.6.0",
+    "leaflet.offline": "^1.0.6",
     "moment": "^2.24.0",
     "msgpack-lite": "^0.1.26",
     "prop-types": "^15.7.2",
-    "rc-slider": "^8.6.9",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-leaflet": "^2.2.1",
-    "react-leaflet-control": "^2.1.1",
-    "react-virtualized": "^9.21.0",
+    "rc-slider": "^8",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-leaflet": "^2.6.1",
+    "react-leaflet-control": "^2.1.2",
+    "react-virtualized": "^9.21.2",
     "serialport": "^7.1.5",
-    "source-map-support": "^0.5.12",
+    "source-map-support": "^0.5.16",
     "xbee-api": "^0.6.0"
   },
   "devDependencies": {
-    "@babel/preset-react": "^7.0.0",
-    "@types/chai": "^4.1.7",
-    "@types/mocha": "^5.2.6",
-    "@types/msgpack-lite": "^0.1.6",
-    "@types/rc-slider": "^8.6.3",
-    "@types/react": "^16.8.16",
-    "@types/react-dom": "^16.8.4",
-    "@types/react-leaflet": "^2.2.1",
-    "@types/react-virtualized": "^9.21.1",
-    "@types/serialport": "^7.0.3",
-    "@typescript-eslint/eslint-plugin": "^1.7.0",
-    "@typescript-eslint/parser": "^1.7.0",
-    "babel-eslint": "^10.0.1",
+    "@babel/preset-react": "^7.8.3",
+    "@types/chai": "^4.2.8",
+    "@types/mocha": "^7.0.1",
+    "@types/msgpack-lite": "^0.1.7",
+    "@types/rc-slider": "^8.6.5",
+    "@types/react": "^16.9.19",
+    "@types/react-dom": "^16.9.5",
+    "@types/react-leaflet": "^2.5.0",
+    "@types/react-virtualized": "^9.21.7",
+    "@types/serialport": "^8.0.0",
+    "@typescript-eslint/eslint-plugin": "^2.19.0",
+    "@typescript-eslint/parser": "^2.19.0",
+    "babel-eslint": "^10.0.3",
     "chai": "^4.2.0",
     "dotenv-webpack": "^1.7.0",
-    "electron": "^3.1.9",
-    "electron-builder": "^20.40.2",
-    "electron-rebuild": "^1.8.4",
-    "electron-webpack": "^2.6.2",
-    "electron-webpack-ts": "^3.1.1",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-import-resolver-typescript": "^1.1.1",
-    "eslint-plugin-chai-friendly": "^0.4.1",
-    "eslint-plugin-import": "^2.17.2",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.13.0",
-    "mocha": "^6.1.4",
-    "stylelint": "^10.0.1",
-    "stylelint-config-standard": "^18.3.0",
-    "stylelint-order": "^3.0.0",
-    "ts-node": "^8.1.0",
-    "typescript": "^3.4.5",
-    "webpack": "^4.30.0"
+    "electron": "^3",
```

Comment:
Leave electron at v3 until we figure out how to add a newer version of it to work with `serialport` and `electron-webpack